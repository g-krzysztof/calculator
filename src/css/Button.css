.Button:last-child {
  margin: 0 0 auto auto;
  width: 149px;
}

.Button__btn {
  width: 74px;
  height: 79px;
  border: none;
  background-color: #fff;
  font-size: 24px;
  cursor: pointer;
}

.Button__btn--score {
  width: 149px;
}
/*# sourceMappingURL=Button.css.map */