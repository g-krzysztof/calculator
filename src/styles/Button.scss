.Button {
    &__btn{
    width: 74px;
    height: 79px;
    border: none;
    background-color: #fff;
    font-size: 24px;
    cursor: pointer;
    }
}