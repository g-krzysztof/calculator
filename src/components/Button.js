import React from 'react';
import '../css/Button.css';

function Button(props) {
    return (
        <div className="Button">
            <button>{props.name}</button>
        </div>
    );
}

export default Button;