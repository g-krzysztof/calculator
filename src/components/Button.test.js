import React from 'react';
import Button from './Button';
import renderer from 'react-test-renderer';

it('renders a Button snapshot', () => {
    const button = renderer.create(<Button name={1} />).toJSON();
    expect(button).toMatchSnapshot();
});