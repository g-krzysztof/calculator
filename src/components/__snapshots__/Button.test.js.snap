// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a Button snapshot 1`] = `
<div
  className="Button"
>
  <button
    className="Button__btn"
    value={1}
  >
    1
  </button>
</div>
`;
